src/code/z_message.c

```diff
@@ -5208,11 +5208,11 @@ void Message_Update(PlayState* play) {
     PauseContext* pauseCtx = &play->pauseCtx;
     InterfaceContext* interfaceCtx = &play->interfaceCtx;
     Input* input = CONTROLLER1(&play->state);
-    s16 averageY;
-    s16 sp50;
+    s16 aveScreenPosY;
```

Comment:
```suggestion
    s16 avgScreenPosY;
```
Perhaps just me, but I'm more use to seeing avg as an abbreviation for average.

Reply:
Actually that's fair. Both seem familiar to me but `avg` does seem better when put side-to-side